apis/testing/conditions.go

```diff
@@ -20,14 +20,12 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	"knative.dev/pkg/apis"
-	duckv1b1 "knative.dev/pkg/apis/duck/v1beta1"
 	"knative.dev/pkg/test"
 )
 
 // CheckCondition checks if condition `c` on `cc` has value `cs`.
-// DEPRECATED: Use versioned test helper
-func CheckCondition(s *duckv1b1.Status, c apis.ConditionType, cs corev1.ConditionStatus) error {
-	cond := s.GetCondition(c)
+func CheckCondition(a apis.ConditionAccessor, c apis.ConditionType, cs corev1.ConditionStatus) error {
```

Comment:
Why not use the condtionManager interface?

Reply:
That interface is huge for what these helpers need

Reply:
secondly we're passing `duck.Status` types here

Reply:
You might also want gettoplevel in the new interface

Reply:
`duck.Status` doesn't have that method so don't want to add it